src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/service/AppealReferenceNumberGeneratorTest.java

```diff
@@ -0,0 +1,120 @@
+package uk.gov.hmcts.reform.iacaseapi.domain.service;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.*;
+import static uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.field.AsylumAppealType.PA;
+import static uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.field.AsylumAppealType.RP;
+
+import java.time.LocalDate;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.hmcts.reform.iacaseapi.domain.DateProvider;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.AppealReferenceNumber;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.field.AsylumAppealType;
+import uk.gov.hmcts.reform.iacaseapi.infrastructure.AppealReferenceNumberInitializerFromCcd;
+
+@RunWith(MockitoJUnitRunner.class)
+@SuppressWarnings("unchecked")
+public class AppealReferenceNumberGeneratorTest {
+
+    private final DateProvider dateProvider = mock(DateProvider.class);
+
+    private final AppealReferenceNumberInitializerFromCcd appealReferenceNumberInitializer =
+            mock(AppealReferenceNumberInitializerFromCcd.class);
+
+    private final AppealReferenceNumberGenerator underTest =
+            new AppealReferenceNumberGenerator(
+                    "50000",
+                    appealReferenceNumberInitializer,
+                    dateProvider);
+
+    @Before
+    public void setUp() {
+        when(dateProvider.now()).thenReturn(LocalDate.now());
+    }
+
+    @Test
+    public void increments_protection_appeal_reference_number() {
+
+        when(appealReferenceNumberInitializer.initialize()).thenReturn(initialReferenceNumbers());
+
+        Optional<String> maybeAppealReferenceNumber =
+                underTest.getNextAppealReferenceNumberFor(PA.getValue());
+
+        assertThat(maybeAppealReferenceNumber.get(), is("PA/50001/2018"));
```

Comment:
not sure, but these tests might have the same problem when moving into 2019 that they'll need refactoring

Reply:
this has been resolved, tests now started with `2018`